src/helm/benchmark/run_specs.py

```diff
@@ -1793,6 +1813,40 @@ def get_wmt_14_spec(language_pair: str, max_train_instances: int = 1) -> RunSpec
     )
 
 
+def get_opinions_qa_spec(
+    survey_type: str,
+    num_logprobs: str,
```

Comment:
Could we rename `num_logprobs` to `num_outputs` everywhere, since it's one less name to have to juggle in the codebase?